core/templates/dev/head/pages/exploration_editor/exploration_save_and_publish_buttons_directive.html

```diff
@@ -69,3 +74,9 @@
     </div>
   </li>
 </ul>
+
+<style>
+  .div-disable {
```

Comment:
This style will apply too generally. We should scope it to this specific directive. 
```suggestion
  exploration-save-and-publish-buttons .div-disable {
```

Reply:
Done